src/core/types/Lab.ts

```diff
@@ -1,3 +1,4 @@
+import { Apy } from '@yfi/sdk';
```

Comment:
Since we re-export SDK needed types from `types/Yearn-Sdk`, you can import directly from '@types'. This way, if someday we need to override the sdk typing on FE, we can do there to propagate through the app.